### ISSUE TITLE
overfitting in temporal stream

### ISSUE THREAD

**Issue by shabu19** [wushidonguc/Two-stream-action-recognition-keras#14]
when I run the temporal-train model, it is overfitting where validation accuracy stagnates at ~50% while training accuracy keeps on increasing.
<img width="893" alt="accuracy" src="https://user-images.githubusercontent.com/34564454/123207201-7242c380-d4d6-11eb-9899-006533a15b17.PNG">
